src/warc2zim/replay/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
```

Comment:
You need to set encoding otherwise the taskbar might be displayed incorrectly if ZIM name uses non-ASCII chars.

Reply:
Notfound and Top as well

Reply:
Good point! Will fix in follow up.